staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go

```diff
@@ -315,23 +320,96 @@ func strategicPatchObject(
 
 type applyPatcher struct {
 	*patcher
+
+	model proto.Schema
```

Comment:
should we add a new file for the applyPatcher & methods?

Reply:
We can. But if we do it should be in a different PR I think

Reply:
That's fine.